src/main/java/core/basesyntax/dao/ReportDao.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.dao;
+
+import java.util.Map;
+
+public interface ReportDao {
+
+    void writeReport(Map<String, Integer> endOfDayFruitQuantities, String filePath);

```

Comment:
See what DAO is
https://en.wikipedia.org/wiki/Data_access_object#:~:text=In%20software%2C%20a%20data%20access,operations%20without%20exposing%20database%20details.